doc/tutorials/CMakeLists.txt

```diff
@@ -33,6 +33,11 @@
 
 # CMakeList for RINGMesh tutorials
 
+set(TUTORIALS_DIRECTORY ${CMAKE_SOURCE_DIR}/doc/tutorials)
+set(TUTORIALS_SOURCE_DIRECTORY ${TUTORIALS_DIRECTORY}/src)
+set(TUTORIALS_DATA_INPUT_DIRECTORY "${TUTORIALS_DIRECTORY}/data")
+set(TUTORIALS_DATA_OUTPUT_DIRECTORY "${PROJECT_BINARY_DIR}/tutorials/output")
+
 if(NOT IS_DIRECTORY ${TUTORIALS_DATA_INPUT_DIRECTORY})
     message(STATUS "Defined data directory for tutorials does not exists, using default doc/tutorials/data")
```

Comment:
Could you use WARNING instead of MESSAGE ?

Reply:
I will remove it, this is not an option anymore.